transform/mattermost-analytics/models/staging/mm_telemetry_prod/stg_mm_telemetry_prod__all.sql

```diff
@@ -0,0 +1,255 @@
+
+with source as (
+    select * from {{ source('mm_telemetry_prod', 'configs') }}
+),
+
+renamed as (
+    select
+        -- Common event columns
+        id               as event_id
+        , event          as event_table
+        , event_text     as event_name
+        , user_id        as server_id
+        , received_at    as received_at
+        , timestamp      as timestamp
+
+        -- Server info
+        , anonymous_id
+        , context_ip as server_ip
+
+         -- OAuth information
+        , config_oauth_enable_gitlab                          as is_gitlab_enabled
+        , config_oauth_enable_google                          as is_google_enabled
+        , config_oauth_enable_office_365                      as is_office365_enabled
+        , config_oauth_enable_openid                          as is_openid_enabled
+        , config_oauth_openid_gitlab                          as is_openid_gitlab_enabled
+        , config_oauth_openid_google                          as is_openid_google_enabled
+        , config_oauth_openid_office_365                      as is_openid_office365_enabled
+
+        -- Ldap information
+        , config_ldap_connection_security                     as connection_security_ldap
+        , config_ldap_enable                                  as enable_ldap
```

Comment:
1/5 Should the `_ldap` prefix kept in all columns?

Reply:
I kept the naming from `stg_mm_telemetry_prod__ldap.sql` (which I believe was based on the old repo naming). If we change it here we'll have to change it there as well. I say, let's keep it as it is and if we see issues/confusions we can change later.